CHANGELOG.md

```diff
@@ -3,6 +3,85 @@
 Full documentation for MIGraphX is available at
 [https://rocmdocs.amd.com/projects/AMDMIGraphX/en/latest/](https://rocmdocs.amd.com/projects/AMDMIGraphX/en/latest/).
 
+## MIGraphX 2.11 for ROCm 6.3.0
+
+### Additions
+
+* Added initial code to run on Windows
+* Added support for gfx120x GPU 
+* Added support for FP8, and INT4
+* Added ONNX operators Bitwise And, Scan,SoftmaxCrossEntropyLoss, GridSample, NegativeLogLikelihoodLoss
+* Added Microsoft Contrib operators MatMulNBits, QuantizeLinear/DequantizeLinear, GroupQueryAttention
+* Added support for the Log2 internal operator
+* Added Split-K as an optional performance improvement
+* Added support for the GCC 14 compiler
+* Added the ablity to call hipBlasLt libaries using environment variable MIGRAPHX_ENABLE_HIPBLASLT_GEMM=1
+* Added scripts to validate ONNX models from the ONNX Model Zoo
+* Enabled the OneHot operator to accept a dynamic batch parameter 
+* Added a --mlir flag the migraphx-driver program to offload entire module to mlir
+* Added GPU Pooling Kernel
+* Added examples for RNNT, and ControlNet
+* Added support for Skip and Simplified LayerNormalization
```

Comment:
Skip Simplified LayerNormalization and Simplied LayerNormalization should be with Microsoft contrib operators